LogicalTypes.md

```diff
@@ -654,13 +654,18 @@ should always be determined by the following rules:
 
 1. If the repeated field is not a group, then its type is the element type and
    elements are required.
-2. If the repeated field is a group with multiple fields, then its type is the
-   element type and elements are required.
-3. If the repeated field is a group with one field and is named either `array`
-   or uses the `LIST`-annotated group's name with `_tuple` appended then the
-   repeated type is the element type and elements are required.
+2. If the repeated field is a group (without `LIST` or `MAP` annotation) with
+   multiple fields, then its type (i.e. a multi-field Tuple) is the element
+   type and elements are required.
+3. If the repeated field is a group (without `LIST` or `MAP` annotation) with
+   one field (with `required` or `optional` repetition) and is named either
+   `array` or uses the `LIST`-annotated group's name with `_tuple` appended,
+   then the repeated type is the element type and elements are required. Note
+   that the element type resolves to a single-field Tuple type.
 4. Otherwise, the repeated field's type is the element type with the repeated
-   field's repetition.
+   field's repetition and elements are required. Note that the element type
+   resolves to List or Map type if the repeated field is `LIST`-annotated or
+   `MAP`-annotated; otherwise the element type resolves to a Tuple type.
```

Comment:
I don't know the worlding of "Tuple type" is better or "Struct type" is better...

Reply:
The example below uses `Tuple` and `OneTuple` and we'd better be consistent.